### PR TITLE
UICONSET-195 UICONSET-198 Add 'Subject source' and 'Subject type' settings to consortium manager

### PR DESCRIPTION
**Pull request by usavkov-epam**
## Purpose

* [UICONSET-195](https://folio-org.atlassian.net/browse/UICONSET-195) View consortium-wide settings for Inventory - Instances - Subject sources.
* [UICONSET-193](https://folio-org.atlassian.net/browse/UICONSET-193) Save consortium-wide settings for Inventory - Instances - Subject sources.
* [UICONSET-194](https://folio-org.atlassian.net/browse/UICONSET-194) Allow user to share Inventory - Instances - Subject sources.
* [UICONSET-198](https://folio-org.atlassian.net/browse/UICONSET-198) View consortium-wide settings for Inventory - Instances - Subject types.
* [UICONSET-196](https://folio-org.atlassian.net/browse/UICONSET-196) Save consortium-wide settings for Inventory - Instances - Subject types.
* [UICONSET-197](https://folio-org.atlassian.net/browse/UICONSET-197) Allow user to share Inventory - Instances - Subject types.

Add 'Subject source' and 'Subject type' settings to consortium manager

## Screenshots

https://github.com/user-attachments/assets/f9955d2c-dc5b-4b29-a81d-8ed6701bcef7

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [ ] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

**Review by SKarolFolio — APPROVED** [review]

**Review by NikitaSedyx — APPROVED** [review]

File: src/routes/ConsortiumManager/settings/inventory/instances/SubjectSources/SubjectSources.test.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import {
+  render,
+  screen,
+} from '@folio/jest-config-stripes/testing-library/react';
+
+import { ConsortiaControlledVocabularyWrapper } from 'helpers';
+import { wrapConsortiaControlledVocabularyDescribe } from 'helpers/wrapConsortiaControlledVocabularyDescribe';
```
Comment (alisher-epam) [inline src/routes/ConsortiumManager/settings/inventory/instances/SubjectSources/SubjectSources.test.js:7]: Can we import it from `'helpers'`?

File: src/routes/ConsortiumManager/settings/inventory/instances/SubjectSources/SubjectSources.test.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      createdDate: '2023-06-26T13:23:49.227+00:00',
+      createdByUserId: '557dbab7-f610-43f1-85b4-c1f9db077959',
+      updatedDate: '2023-06-26T13:23:49.227+00:00',
+      updatedByUserId: '557dbab7-f610-43f1-85b4-c1f9db077959',
+    },
+  },
+];
+
+wrapConsortiaControlledVocabularyDescribe({ entries })('SubjectSources', () => {
+  it('should render controlled vocabulary list with subject sources', async () => {
```
Comment (alisher-epam) [inline src/routes/ConsortiumManager/settings/inventory/instances/SubjectSources/SubjectSources.test.js:46]:
```suggestion
  it('should render controlled vocabulary list with subject sources', () => {
```

File: src/routes/ConsortiumManager/settings/inventory/instances/SubjectTypes/SubjectTypes.test.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
+import {
+  render,
+  screen,
+} from '@folio/jest-config-stripes/testing-library/react';
+
+import { ConsortiaControlledVocabularyWrapper } from 'helpers';
+import { wrapConsortiaControlledVocabularyDescribe } from 'helpers/wrapConsortiaControlledVocabularyDescribe';
```
Comment (alisher-epam) [inline src/routes/ConsortiumManager/settings/inventory/instances/SubjectTypes/SubjectTypes.test.js:7]: The same here as above

File: src/routes/ConsortiumManager/settings/inventory/instances/SubjectTypes/SubjectTypes.test.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+      createdDate: '2023-06-26T13:23:49.227+00:00',
+      createdByUserId: '557dbab7-f610-43f1-85b4-c1f9db077959',
+      updatedDate: '2023-06-26T13:23:49.227+00:00',
+      updatedByUserId: '557dbab7-f610-43f1-85b4-c1f9db077959',
+    },
+  },
+];
+
+wrapConsortiaControlledVocabularyDescribe({ entries })('SubjectTypes', () => {
+  it('should render controlled vocabulary list with subject types', async () => {
```
Comment (alisher-epam) [inline src/routes/ConsortiumManager/settings/inventory/instances/SubjectTypes/SubjectTypes.test.js:44]:
```suggestion
  it('should render controlled vocabulary list with subject types', () => {
```

**Pull request merged**